src/modules/inspect/inspect-correlation/inspect-correlation.scss

```
@@ -68,6 +68,7 @@
 }
 
 .inspect-correlation__token-viewer {
+  height: 100%;
```

This would cause the resizing of the inspect panel to lag.

```suggestion
  height: 100%;
  overflow: hidden;
```